### ISSUE TITLE
[Feature Request] Locally build documentation for API preview

### ISSUE THREAD

**Issue by Benjamin-eecs** [facebookresearch/rl#287]
## Motivation

To fully understand the API of torchrl, maybe it can be a temporal solution for users to build docs locally to review the API

## Solution

A `make doc` command like `cd docs && make html && cd _build/html && python3 -m http.server`

## Checklist

- [x] I have checked that there is no similar issue in the repo (**required**)

**Comment by vmoens** [comment]
You're totally right, @nicolas-dufour suggested the same. The doc needs some cleaning but once it's done we'll take care of this.
On the long term the doc will sit on PyTorch (hopefully during next fall) so we're not considering using a dedicated domain or readthedocs.
